octree_web_viewer/client/index.html

```diff
@@ -14,17 +14,18 @@
   limitations under the License.
 -->
 <!doctype html>
+
 <head>
 </head>
+
 <body>
   <div id="renderArea" style="position: absolute; width: 100%; height: 100%; overflow: hidden;"></div>
 
   <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.5.1/dat.gui.min.js"></script>
   <script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/r82/three.min.js"></script>
```

Comment:
The version of three.js is wrong here, could you please change it to point to the latest?

Reply:
As discussed offline, we can remove those 2 lines altogether. Thanks for bringing this up.